lib/extensions.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+const path = require('path');
+
+module.exports = {
+
+  withConflictExclusions: function(BaseGenerator) {
```

Comment:
We can call it enhanced conflicted and by that cover all possible future changes \ additions

Reply:
Please rename it to with enhancedConflicted